commands/create_cluster.go

```diff
@@ -23,11 +23,11 @@ type cpuDefinition struct {
 }
 
 type memoryDefinition struct {
-	SizeGB int `yaml:"size_gb,omitempty"`
+	SizeGB float32 `yaml:"size_gb,omitempty"`
 }
 
 type storageDefinition struct {
-	SizeGB int `yaml:"size_gb,omitempty"`
+	SizeGB float32 `yaml:"size_gb,omitempty"`
```

Comment:
This is a personal thing. Would like to use `float64` for simplicity. We do not need to count bytes in the client. The API also uses `float64`. I feel like `float64` is just more common when you do not care about memory allocation.